infra/modules/azure_monorepo_single_env_starter_pack/variables.tf

```diff
@@ -0,0 +1,75 @@
+variable "tags" {
+  type        = map(any)
+  description = "Resources tags"
+}
+
+variable "environment" {
+  type = object({
+    prefix          = string
+    env_short       = string
+    location        = string
+    domain          = string
+    instance_number = string
+  })
+
```

Comment:
```suggestion
validation {
    condition     = can(regex("^[a-z]+$", var.environment.prefix))
    error_message = "Prefix must contain only lowercase letters."
  }

  validation {
    condition     = length(var.environment.env_short) <= 3
    error_message = "Environment short name must be 3 characters or less."
  }
```

Reply:
These values are already validated in `azurerm_naming_convention` module